theano/sandbox/rng_mrg.py

```diff
@@ -285,11 +286,17 @@ def mrg_next_value(rstate, new_rstate):
 class mrg_uniform_base(Op):
     # TODO : need description for class, parameter
     __props__ = ("output_type", "inplace")
+    params_type = ParamsType(inplace=bool_t, ndim=int_t, otypenum=int_t, otype_is_float32=bool_t)
```

Comment:
ndim don't need to be a param. It can be taken from the input at run time.

PyArray_NDIM(...)

Similar for otype_is_float32